main.js

```diff
@@ -481,23 +506,36 @@ app.post('/create-user', upload.single('avatarURL'), async function (req, res) {
 
 	try {
 		// Default values
-		const uploadedAvatar = {
-			data: req.file.buffer,
+		const uploadedAvatar = req.file ? {
+			data: {
+				$binary: {
+					base64: req.file.buffer
+				}
+			},
 			contentType: req.file.mimetype
+		} : undefined;
+		const defaultProfilePicture = await readFile("./public/images/Default-Profile-Picture.txt", "utf-8");
+		const defaultAvatar = {
+			data: {
+				$binary: {
+					base64: defaultProfilePicture
+				}
+			},
+			contentType: 'image/jpeg'
 		}
-		const defaultAvatar = '/images/Default-Profile-Picture.jpg';
 		const defaultAchievements = ["gettingStarted", "planKit", "finishKit"];
-
+		const defaultKits = [];
 		/** @type {UserDoc} */
 		const newUserDoc = {
 			_id: req.body?._id ?? undefined,
 			name: req.body?.name ?? undefined,
 			emailAddress: req.body?.emailAddress ?? undefined,
 			pwd: req.body?.pwd ?? undefined,
 			admin: (req.body?.admin == 'true') ? true : false, // Doing this because formdata can't send booleans
```

Comment:
```suggestion
			admin: (req.body?.admin == 'true'), // Doing this because formdata can't send booleans
```
`? true : false` is a no-op. 😄